### ISSUE TITLE
Drawer is not working as expected in RTL

### ISSUE THREAD

**Issue by baselbj** [react-navigation/react-navigation#5279]
### Current Behavior

in LTR Drawer is working fine but in RTL it has space at the same drawer position.

**left:**
![screenshot_20181124-181656](https://user-images.githubusercontent.com/12442925/48971210-4eb5f480-f01f-11e8-8c89-1453e4fb85ec.png)

**right:**
![screenshot_20181124-181743](https://user-images.githubusercontent.com/12442925/48971229-90469f80-f01f-11e8-9934-1b085190474a.png)

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 3.0.0
| react-native     | 0.57.4
| node             | 11.0.0
| npm or yarn      | yarn 1.12.1

**Comment by brentvatne** [comment]
Post a reproducible example please

**Comment by baselbj** [comment]
@brentvatne thank you for your response :) 
the issue appers when making app RTL ready as discribed [here](https://facebook.github.io/react-native/blog/2016/08/19/right-to-left-support-for-react-native-apps#making-an-app-rtl-ready)

this will show a space as shown in the screenshot attached.

**Comment by brentvatne** [comment]
@baselbj - can you post a MCVE please? (https://stackoverflow.com/help/mcve)

**Comment by MoudiZd** [comment]
@baselbj 

I solve that by hard writing a new width property iand backgroundColor nside this file:

[react-navigation-drawer/src/views/DrawerLayout.js](url)

within styles component for the 'drawerContainer' element

  ```
drawerContainer:  {
...StyleSheet.absoluteFillObject,
    zIndex: 1001,
    flexDirection: 'row',
    +backgroundColor: '#333333',
    +width: Dimensions.get('window').width - 150
  }
```
    
I know this is not professional but i don't have time to wait until this issue is solved and don't have time to pull a request because i don't know if this solution is suitable for all cases but it is sufficient for me now

Hope this issue will be solved soon

**Comment by brentvatne** [comment]
@MoudiZd - post a MCVE please: https://stackoverflow.com/help/mcve

**Comment by MoudiZd** [comment]
@brentvatne  I posted it here : [https://stackoverflow.com/questions/53490486/react-navigation-drawer-rtl-issue](url) please take a look

Thank you for any regards

**Comment by MoudiZd** [comment]
@brentvatne 
this is the repository for reproducing the issue

[https://github.com/MoudiZd/react-navigation-rtl](url)

**Comment by MoudiZd** [comment]
I created a new repository for that and that is the link i hope now that everything is clear enough

[https://github.com/MoudiZd/react-navigation-rtl](url)

**Comment by MoudiZd** [comment]
@baselbj

Please update the title of your issue to become

Drawer is *not* working as expected in RTL

Best regards

**Comment by MoudiZd** [comment]
@baselbj 
Have you tried my workaround situated here ?

[https://github.com/react-navigation/react-navigation/issues/5279#issuecomment-441821092](https://github.com/react-navigation/react-navigation/issues/5279#issuecomment-441821092)

**Comment by baselbj** [comment]
@MoudiZd thank you I will try finding a solution depending on your workaround

**Comment by brentvatne** [comment]
thanks for that @MoudiZd, very helpful. this is fixed in react-navigation@3.0.2
